crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -637,7 +689,7 @@ impl SpecializedMeshPipeline for MeshPipeline {
             // For the opaque and alpha mask passes, fragments that are closer will replace
             // the current fragment value in the output and the depth is written to the
             // depth buffer
-            depth_write_enabled = true;
+            depth_write_enabled = !key.contains(MeshPipelineKey::PREPASS_DEPTH);
```

Comment:
this always evaluates to true, the PREPASS_DEPTH flag is only set when rendering the prepass itself. i think it's fine to leave it as true (it may be more efficient not to, but maybe things that don't write depth in the prepass still should write depth in the main pass..?) but let's make it explicit